### PR TITLE
Request transfer

### PR DESCRIPTION
**Pull request by jiaxuan1812**
## Preparations
Please check if the PR fulfills these requirements
- [x] Readme (How to Install, Run, Use and Configure) wurde aktualisiert falls nötig
- [x] Das Feature ist dokumentiert
- [x] Alle nicht getter und setter Methoden sind dokumentiert
- [x] Es wurden Unit tests für neue Features erstellt
- [ ] Alle Tests müssen erfolgreich sein
- [x] Mindestens ein Reviewer muss den Pull Request akzeptieren

## :ticket: Description

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This PR realizes an essential part of the chain request use case wherein a request cascades through multiple services - in our case three services contribute in this scenario - with the account service initiating the request chain by sending a request to the price service at url "localhost:3300/request/account-worth" which subsequently transfers it to the database service at endpoint "/account-worth". The response of the database service is processed and handled correspondingly in the price service before a response is emitted to the originator of the request chain, the account service. The pre-existing circuit breaker logic is naturally applied to the novel request, signalizing undesired behaviour in form of errors that consequently undergo assessment and are reflected in the respective response. Furthermore, upon the success of a request, the fetched data is now enfolded in the response, providing more insight into the request itself. 

* **What is the current behavior?** (You can also link to an open issue here)
The price service as of right now before the merge of the pull request neither bears the endpoint /account-worth nor the logic for the chained request use case. The latter includes the evaluation of the response sent back by the database service and needless to say, the transfer to the database service itself. It still encapsulates the circuit breaker logic for all the request it sends and the response does not yet comprise the fetched data.

* **What is the new behavior (if this is a feature change)?**
With this new feature, the price service sees the realization of the chained request use case in form the addition of a novel router handler the function responsible for transferring the received request to the database service. Moreover, the respective response handling has been rendered applicable to the chain request use case with the fetched data now being an inherent part of the response of a successful request reception. 

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Most changes include valuable additions for which users do not necessarily have to change their application drastically. What is seminal however, is the novel property "result" in the response object upon a successful request which holds the fetched data from the request, and the possibility of querying a fixed number representing a simulated account worth at url "localhost:3300/request/account-worth" in the price service. 

* **Other information**:

## :lock: Closes
Which issue(s) is (are) being closed by the PR?
The Jira-Ticket "SLADIM-78" is closed with this feature addition.

### CODE REVIEW

File: priceservice/src/app.controller.ts [hunk 1]
```diff
@@ -22,15 +24,29 @@ export class AppController {
...
-    this.configHandlerService.databaseUrl ='http://localhost:3000/request-handler/customer-name';
+    this.configHandlerService.databaseUrl =
+      'http://localhost:3000/request-handler/customer-name';
+    return this.appService.handleRequest();
+  }
+
+  /**
+   * The request sent by the account service will go to this endpoint and
+   * will be transferred to the database service. A timeout interceptor is also
+   * bind to this router handler that will throw a timeout exception after 5 seconds.
```
Comment (stiesssh) [inline priceservice/src/app.controller.ts:44]: bind -> bound
passiv wird mit past participle gebildet ;)

Comment (jiaxuan1812) [reply priceservice/src/app.controller.ts:44]: Danke 👍 DIe Doku von den vorherigen Funktionen habe ich mir nicht mehr so genau angeschaut :D

File: priceservice/src/app.service.ts [hunk 2]
```diff
@@ -1,136 +1,220 @@
...
+      .subscribe(
+        res =>
+          console.log(
+            `Report sent to monitor at ${this.configHandlerService.monitorUrl}`,
+          ),
+        err =>
+          console.log(
+            `Monitor at ${this.configHandlerService.monitorUrl} not available`,
+          ),
+      );
```
[the remark this answers is not in the document]
Comment (jiaxuan1812) [reply priceservice/src/app.service.ts:105]: Ja ich meine schon. Error handling halt ich weiss nicht wie es anders implementiert werden sollte.

[the remark this answers is not in the document]
Comment (MarcelWolkober) [reply priceservice/src/app.service.ts:108]: Also was die Tests angeht: bei mir schlagen die immer fehl, irgendwas stimmt bei mir in der Konfiguration nicht. Vielleicht versuchst du mal noch nen einfachen test hinzu zufügen, und dann kann man es auch eigentlich Mergen.

[the remark this answers is not in the document]
Comment (jiaxuan1812) [reply priceservice/src/app.service.ts:108]: Bei welcher Konfiguration?

[the remark this answers is not in the document]
Comment (MarcelWolkober) [reply priceservice/src/app.service.ts:108]: Hab die gleichen Fehler die auch von den Automated-Tests geworfen werden, aber keine Ahnung wie ich das fixen kann.

**Pull request merged**